core/auth/oauth/oidc.go

```diff
@@ -467,7 +467,15 @@ func (i *openIDIdentifier) Callback(ctx context.Context, request *web.Request, r
 		return i.responder.ServerError(err)
 	}
 
-	oauth2Token, err := oauthConfig.Exchange(ctx, code)
+	options := make([]oauth2.AuthCodeOption, 0, len(i.authcodeOptions))
+	for _, o := range i.authcodeOptions {
```

Comment:
we should drop this for loop since this is only used for setting the scope param which is not needed in the token endpoint call..